oxidation/libparsec/crates/core_fs/src/storage/chunk_storage.rs

```diff
@@ -485,8 +487,8 @@ mod tests {
         );
 
         let local_chunk_ids = chunk_storage.get_local_chunk_ids(&chunk_ids).await.unwrap();
-        let set0: HashSet<_, RandomState> = HashSet::from_iter(local_chunk_ids.iter());
-        let set1: HashSet<_, RandomState> = HashSet::from_iter(chunk_ids.iter());
+        let set0: HashSet<_> = HashSet::from_iter(local_chunk_ids.iter());
+        let set1: HashSet<_> = HashSet::from_iter(chunk_ids.iter());
```

Comment:
same

Reply:
done

Note: You could have a one comment for the 2 lines by selecting them